### ISSUE TITLE
Caffe2 support?

### ISSUE THREAD

**Issue by alice-smith** [beniz/deepdetect#291]
Now that Caffe2 has been released, are there any plans to integrate that with DD?

**Comment by beniz** [comment]
Absolutely, and for some time now :) We're still reviewing Caffe2's C++ API.

**Comment by beniz** [comment]
Use C++ caffe2 unofficial tutorial with some examples, https://github.com/leonardvandriel/caffe2_cpp_tutorial

**Comment by beniz** [comment]
See #424
